pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -145,6 +150,21 @@ type CertificateSpec struct {
 	KeyEncoding KeyEncoding `json:"keyEncoding,omitempty"`
 }
 
+type X509DistinguishedName struct {
+	// Country/Region
+	Country            []string `json:"country,omitempty"`
+	Organization       []string `json:"organization,omitempty"`
+	OrganizationalUnit []string `json:"organizationalUnit,omitempty"`
+	// City
+	Locality []string `json:"locality,omitempty"`
+	// State/Province
+	Province      []string `json:"province,omitempty"`
+	StreetAddress []string `json:"streetAddress,omitempty"`
+	PostalCode    []string `json:"postalCode,omitempty"`
+	SerialNumber  string   `json:"serialNumber,omitempty"`
+	CommonName    string   `json:"commonName,omitempty"`
```

Comment:
Please also remove this field - we have `spec.commonName` already and if we do want to move that field, it'll have to happen as part of v1alpha3